### PR TITLE
Updating to not use global typings

### PR DESCRIPTION
**Pull request by rorticus**
Using the latest `@dojo/shim` and `@dojo/core` and removing the symbol shim typings.

### CODE REVIEW

**Comment by agubler** [comment]
Can you do it for todo while you're at it.

**Comment by rorticus** [comment]
@agubler , updated to include `todo-mvc`!

**Comment by rorticus** [comment]
Good tip @agubler ! I removed the `typings.json` file now and also cleaned up a bunch of old unused packages from `package.json`. Also, updated to latest packages on all the `@dojo/` stuff.

File: todo-mvc/typings.json [hunk 1]
```diff
@@ -1,8 +1,5 @@
 {
 	"name": "dojo2-todo-mvc",
-	"globalDependencies": {
-		"symbol-shim": "github:dojo/typings/custom/symbol-shim/symbol-shim.d.ts#003aa24b3448804a2c59e9a1d9740ca56e067e79"
-	},
 	"globalDevDependencies": {
```
Comment (agubler) [inline todo-mvc/typings.json]: should be able to remove this.

Comment (rorticus) [inline todo-mvc/typings.json]: Removed!

**Pull request merged**